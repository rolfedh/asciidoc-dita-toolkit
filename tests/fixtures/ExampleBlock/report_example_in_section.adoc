// Unsupported examples in sections:
====
An unsupported example.
====


[example]
An unsupported example.

== Section title


