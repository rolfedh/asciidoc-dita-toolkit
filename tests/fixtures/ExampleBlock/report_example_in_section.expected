// Unsupported examples in sections:

[example]
An unsupported example.

====
An unsupported example.
====

== Section title
